keyboards/zsa/voyager/info.json

```diff
@@ -0,0 +1,204 @@
+{
+    "manufacturer": "ZSA Technology Labs",
+    "keyboard_name": "Voyager",
+    "maintainer": "ZSA Technology Labs",
+    "url": "zsa.io/voyager",
+    "processor": "STM32F303",
+    "bootloader": "stm32-dfu",
```

Comment:
For the `stm32-dfu` bootloader you can override the `bootloader_jump()` and `mcu_reset()` implementations, but `enter_bootloader_mode_if_requested()` with the code to jump to the system memory bootloader still remains.  So you may actually need to declare the bootloader type as `custom` to remove all code related to `stm32-dfu`.

Reply:
Yeah, custom seems to be the better option. And was able to get it to call dfu-util "well".